src/menu.json

```diff
@@ -118,7 +118,6 @@
   },
   {
     "categoryName": "Summer University",
-    "auth": "logged-in",
```

Comment:
Why removing this?

Reply:
SUs are public, so it is nice that people can see it in the sidebar. The other components are still blocked by either logged-in or permissions